pkg/sarif_test.go

```diff
@@ -24,9 +24,43 @@ import (
 	"go.uber.org/zap/zapcore"
 
 	"github.com/ossf/scorecard/v2/checker"
-	"github.com/ossf/scorecard/v2/docs/checks"
 )
 
+func sarifMockDocRead() (*mockDoc, error) {
```

Comment:
Why declare this as a global var instead of per testcase as it was before?

Reply:
It simplifies the test. The per-testcase ones were identical to one-another. I should have made them global from the beginning :(